iceoryx_utils/include/iceoryx_utils/internal/posix_wrapper/unix_domain_socket.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
This is work in progress. First functionality already in this PR but not finalized and not yet used